benchto-service/src/main/webapp/js/controllers.js

```diff
@@ -243,6 +243,95 @@
                     return benchmarkRun.sequenceId;
                 }).join();
                 $location.path('compare/' + names + '/' + sequenceIds);
+            };
+
+            $scope.exportCsv = function () {
+                var names = _.map($scope.compareBenchmarkRuns, function (benchmarkRun) {
+                    return benchmarkRun.uniqueName;
+                });
+                var sequenceIds = _.map($scope.compareBenchmarkRuns, function (benchmarkRun) {
+                    return benchmarkRun.sequenceId;
+                });
+
+                $scope.benchmarkRuns = [];
+                for (var i in sequenceIds) {
+                    BenchmarkService.loadBenchmarkRun(names[i], sequenceIds[i])
+                        .then(function (benchmarkRun) {
+                            $scope.benchmarkRuns.push(benchmarkRun);
+                            prepareCsv();
+                        });
+                }
+                console.log("x " + $scope.benchmarkRuns );
+
+                var csvDataDefer = $q.defer();
+                var prepareCsv = function () {
+                    if (sequenceIds.length != $scope.benchmarkRuns.length) {
+                        return; // not all benchmarkRuns are loaded yet
+                    }
+
+                    $scope.benchmarkRuns = _.sortBy($scope.benchmarkRuns, function(run) {
+                       return run.uniqueName + run.sequence_id;
+                    })
+
+                    var benchmarkRunsHelper = new BenchmarkRunsHelper($scope.benchmarkRuns, []);
+                    //var aggregatedMeasurementsKeys = benchmarkRunsHelper.aggregatedExecutionsMeasurementKeys();
+                    var aggregatedMeasurementsKeys = ['duration']; // let's stick to duration for now
```

Comment:
I took just duration as @pnowojski states that other measurements are useless and do not correspond to reality.